pkg/dedicatedadmin/dedicatedadmin_test.go

```diff
@@ -42,3 +45,65 @@ func TestBlacklist(t *testing.T) {
 		}
 	}
 }
+
+func TestExpectedBlacklist(t *testing.T) {
+	var configfilename = "../../manifests/50-dedicated-admin-operator-config.ConfigMap.yaml"
```

Comment:
I'm not sure what the benefit of this is. The unique thing the dedicatedadmin package is doing is splitting a string on `,`. Looping over the resulting slice and calling `regexp.MatchString` is well-tested by upstream Go, thus we don't need to test the loop runs `n` times, or that `regexp.MatchString` is behaving [as documented](https://golang.org/pkg/regexp/#Regexp.MatchString).

We can provide fixture data (to `IsBlackListedNamespace`) that is divorced from the live data and intentionally cause breakages in code that _we're_ responsible for. That is, we can provide, `^[kube-.*` and demonstrate a failure to match `kube-system` (because of the `[`); we would never discover this until a typo was included in the `manifests` config file because this test assumes clean input.

So, then, what _are_ the edge cases? A malformed input string, either because of a failure to correctly split on `,`, or because the regular expression doesn't [compile](https://golang.org/pkg/regexp/#Compile). These cases don't particularly care about the distributed config in `manifests`.

I hope with a smaller changeset this comment makes more sense than in the context of the larger changeset (https://github.com/openshift/dedicated-admin-operator/pull/13).